src/coreclr/tools/Common/TypeSystem/Common/MethodDesc.cs

```diff
@@ -20,10 +20,10 @@ public enum MethodSignatureFlags
         UnmanagedCallingConventionThisCall   = 0x0003,
         CallingConventionVarargs             = 0x0005,
         UnmanagedCallingConvention           = 0x0009,
+        AsyncCallingConvention               = 0x0100,
 
         Static = 0x0010,
         ExplicitThis = 0x0020,
-        AsyncCallConv = 0x0040,
```

Comment:
Keep the AsyncCall here, maybe even separate with new line. It does not logically belong together with `UnmanagedCallingConvention` bits.